website/content/posts/introducing-action-validate-image.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Introducing Action Validate for GitHub"
+date: 2023-10-24T13:02:00-04:00
+author: "Sean Conroy"
+---
+<!--more-->
+
+Introducing Action Validate you may already be familiar with using the `EC-CLI Validate` command for local container image validation. Now, you can seamlessly integrate this functionality directly into your build processes or any other automated workflow in GitHub.
+
+## What is EC Validate Action?
+
+**EC Validate** is a GitHub Action that brings validation capabilities of the Enterprise Contract CLI directly into your GitHub Workflow. Designed to integrate seamlessly into your build pipeline, this action offers both compliance assessments and integrity checks for your container images. It ensures that your images meet both organizational and industry standards before deployment or environment promotion.
+
+### Key Features
+
+- **GitHub Native**: Being a GitHub Action, EC Validate seamlessly integrates into your existing GitHub workflows, while also providing GitHub summary output.
+- **Policy Compliance**: Ability to tailor its validation checks based on custom or pre-defined policies.
+- **Integrity Checks**: Verifies that the image hasn't been tampered with. 
+- **[Signature Support](https://enterprisecontract.dev/docs/ec-cli/main/signing.html)**: Offers support for both Long-Lived Public-Key Authentication and Keyless Authentication.
```

Comment:
I don't think we need all the capitalization here.

Also some wording change suggestions.

```suggestion
- **[Signature Verification Support](https://enterprisecontract.dev/docs/ec-cli/main/signing.html)**: Offers support for verifying both long-lived public-key signed, and keyless signed container images.